ansible/playbooks/roles/preflight/tasks/main.yml

```diff
@@ -30,6 +30,36 @@
 
 - include_tasks: check-os.yml
 
+- name: Check routing configuration
+  shell: ip route | grep ^default | wc -l
```

Comment:
* `set -o pipefail`
* `wc -l` can be removed and instead `length` filter + `stdout_lines` could be used
* may be something different can be used - https://serverfault.com/a/731095/426313